Omeprazole/Omeprazole_evaluation_report.md

```diff
@@ -413,19 +413,19 @@ The first plot shows observed versus simulated plasma concentration, the second
 
 ![002_plotGOFMergedResidualsOverTime.png](images/003_3_Results_and_Discussion/002_3_2_Diagnostics_Plots/002_plotGOFMergedResidualsOverTime.png)
 
-GMFE = 1.503354 
+GMFE = 1.503352 
 
 ![004_plotGOFMergedPredictedVsObserved.png](images/003_3_Results_and_Discussion/002_3_2_Diagnostics_Plots/004_plotGOFMergedPredictedVsObserved.png)
 
 ![005_plotGOFMergedResidualsOverTime.png](images/003_3_Results_and_Discussion/002_3_2_Diagnostics_Plots/005_plotGOFMergedResidualsOverTime.png)
 
-GMFE = 2.395866 
+GMFE = 2.395867 
 
 ![007_plotGOFMergedPredictedVsObserved.png](images/003_3_Results_and_Discussion/002_3_2_Diagnostics_Plots/007_plotGOFMergedPredictedVsObserved.png)
 
 ![008_plotGOFMergedResidualsOverTime.png](images/003_3_Results_and_Discussion/002_3_2_Diagnostics_Plots/008_plotGOFMergedResidualsOverTime.png)
 
-GMFE = 2.374414 
+GMFE = 2.501750 

```

Comment:
This is the only place among all models with the significant change of GMFE @pavelbal

Reply:
Very low values are treated differntly again, this was already the case between v9 and v10. See time-concentration profile:

V10:
https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library/blob/develop/Omeprazole/images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/002_3_3_2_Model_Verification/015_plotTimeProfile.png

V11:
https://github.com/Yuri05/OSP-PBPK-Model-Library/blob/OSP11_140_128/Omeprazole/images/003_3_Results_and_Discussion/003_3_3_Concentration-Time_Profiles/002_3_3_2_Model_Verification/015_plotTimeProfile.png

Nothing to do from my perspective.